src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -27,6 +27,13 @@
                 Description="Produce a package file during build operations."
                 Category="General" />
 
+  <!-- TODO add a fwlink for this help URL -->
+  <BoolProperty Name="IsPublishable"
+                DisplayName="Allow project publishing (Publish target)"
+                Description="When publishing outside of Visual Studio, this allows commands such as &quot;dotnet publish&quot; to execute on this project."
```

Comment:
`DisplayName` is the heading and `Description` is the check box's label. Most bool properties have a description which is more directive. For example:

```suggestion
                DisplayName="Publishing"
                Description="Allow publishing outside of Visual Studio, such as via &quot;dotnet publish&quot;."
```

Reply:
I based the text on the Bool property above it, which looks like:
```xml
  <BoolProperty Name="GeneratePackageOnBuild"
                DisplayName="Generate NuGet package on build"
                Description="Produce a package file during build operations."
                Category="General" />
```
I remembered the 'action verb' style from previous conversations, but I applied it to the DisplayName and not the Description. :-(

Reply:
Updated the text.